### PR TITLE
New Chem, Frigorific Mixture 

### PR DESCRIPTION
**Pull request by LaOubliette**
Allows ghetto chemists to cool mixtures with salty ice water. 

Currently, Cryostylane allows you to also do this but is practically useless as it is only easily made in a lab setting where heating/cooling mixtures is done by a roundstart machine. 

# Wiki Documentation
Frigorific Mixture is made with 1 part sodium-chloride and 1 part ice. It reacts with water rather than oxygen, but most other variables from cryostylane can be put in its description. 

# Changelog
:cl:  
rscadd: Added salty ice water that makes mixtures cold

/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm [hunk 1]
```diff
@@ -469,3 +469,23 @@
...
 	required_temp = 200
 	is_cold_recipe = TRUE
+
+/datum/chemical_reaction/frigorific_mixture
+	name = /datum/reagent/frigorific_mixture
+	id = /datum/reagent/frigorific_mixture
+	results = list(/datum/reagent/frigorific_mixture = 2)
+	required_reagents = list(/datum/reagent/consumable/sodiumchloride = 1, /datum/reagent/consumable/ice = 1)
+
+/datum/chemical_reaction/cryostylane/on_reaction(datum/reagents/holder, created_volume)
```
Comment (Moltijoe) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: cryostlane?

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm (hunk 1 above)
Comment (LaOubliette) [reply code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: ah i missed one mb

Comment (LaOubliette) [reply code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: should be fine now

**Review by Moltijoe — CHANGES_REQUESTED** [review]
sql, bruh what?

File: SQL/tgstation_schema_prefixed.sql
Comment (ynot01) [inline SQL/tgstation_schema_prefixed.sql:1]: remove changes

**Review by ynot01 — CHANGES_REQUESTED** [review]

**Comment by MajManatee** [comment]
Reopen when work continutes

**Pull request closed without merging**